### PR TITLE
Fix root url

### PR DESCRIPTION
**Pull request by marceloschmidt**
Fix using ROOT_URL within a subfolder.
Add routes for simplified URLs: /repo/branch/alias, /branch/alias and /alias (with or without the ROOT_URL subfolder)/

### CODE REVIEW

**Comment by mikemurray** [comment]
Some console errors.

screen shot 2016-03-24 at 2 48 40 pm

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "react-select": "^1.0.0-beta8",
     "sha1": "^1.1.1",
     "underscore": "^1.8.3",
-    "underscore.string": "^3.3.4"
+    "underscore.string": "^3.3.4",
+    "url": "^0.11.0""
```
Comment (aaronjudd) [inline package.json]: There is an extra **breaking** quote here.

**Comment by aaronjudd** [comment]
How will this handle multiple repos with the same branch / structure? ie: two repos with `master/tags`?

**Comment by zenweasel** [comment]
No response, so closing for now. Please reopen if you correct the issues.

**Comment by marceloschmidt** [comment]
Yes, I should have made a few more tests... I'm merging redoc changes into our repository and will reopen if we think it's good for everyone ;) Sorry, and thank you!

**Pull request closed without merging**